client/my-sites/plan-card/item.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+
+export default React.createClass( {
+
+	displayName: 'PlanCardItem',
+
+	propTypes: {
+		highlight: PropTypes.bool,
+		strikethrough: PropTypes.bool
+	},
+
+	getDefaultProps() {
+		return {
+			highlight: false,
+			strikethrough: false
```

Comment:
Should we use a more semantic name less related to the visual treatment? Like `is-unavailable`.